.github/workflows/add-lockdown-label.yml

```diff
@@ -0,0 +1,78 @@
+name: Add Branch Lockdown Label to PRs
+
+on:
+  schedule:
+    # Run daily at 00:00 UTC
+    - cron: '0 23 * * *'
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+permissions:
+  actions: write # For managing the operation state cache
+  issues: write
+
+jobs:
+  add-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install jq
+        run: sudo apt-get install -y jq
+
+      - name: Add label to PRs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Determine the third Tuesday of the current month
+          third_tuesday=$(date -d "$(date +%Y-%m-01) +14 days" +%Y-%m-%d)
+          while [ $(date -d "$third_tuesday" +%u) -ne 2 ]; do
+            third_tuesday=$(date -d "$third_tuesday + 1 day" +%Y-%m-%d)
+          done
+          
+          # Determine the first Tuesday of the current month
+          first_tuesday=$(date -d "$(date +%Y-%m-01)" +%Y-%m-%d)
+          while [ $(date -d "$first_tuesday" +%u) -ne 2 ]; do
+            first_tuesday=$(date -d "$first_tuesday + 1 day" +%Y-%m-%d)
+          done
+
+          # Get current date
+          current_date=$(date +%Y-%m-%d)
+
+          echo "Current Date: $current_date"
+          echo "Third Tuesday of the month: $third_tuesday"
+          echo "First Tuesday of the month: $first_tuesday"
+
+          # Check if the current date is after the third Tuesday of this month or before the first Tuesday of this month
+          if [[ "$current_date" > "$third_tuesday" || "$current_date" < "$first_tuesday" ]]; then
+            echo "Within the label period. Adding labels to PRs..."
+
+            # Get all open PRs created by app/dotnet-maestro and targeting branches release/8* and release/9* excluding release/9.0.3xx
```

Comment:
So, our branch-to-branch codeflow PRs use `app/github-actions`. Also, why are we limiting the PRs based on who created them? If the branch is locked down, *all PRs* targeting those branches should have "Branch Lockdown", right?

Reply:
At least today, I use branch lockdown also when the branch unlocks to find all PRs that should go in. If I don't limit this, I'll pick up other PRs that just happened to target those branches. Not a good excuse and I think your argument to just tag all of them is probably stronger.